src/components/report/Color.js

```diff
@@ -3,7 +3,9 @@ export default {
 	props: ["font"],
 	data() {
 		return {
-			palettes: this.font.colorPalettes
+			palettes: this.font.colorPalettes,
+			hasPalettes:
+				this.font.colorPalettes && this.font.colorPalettes.length !== 0
```

Comment:
`colorPallettes` always returns an array, right? So you don't have to check for it to be non null?

Reply:
I ran into this breaking when I was using the old engine, that didn't return `colorPalettes` at all. So I got a bit too precocious here perhaps.

Reply:
Turned back!